tests/e2e/specs/shopper/mini-cart.test.js

```diff
@@ -221,6 +234,90 @@ describe( 'Shopper → Mini Cart', () => {
 		} );
 	} );
 
+	describe( 'Tax included', () => {
+		let taxSettings;
+		beforeAll( async () => {
+			taxSettings = ( await WooCommerce.get( 'settings/tax' ) ).data;
+			/**
+			 * Set the tax display settings to show prices including tax during
+			 * cart and checkout. The price displayed in the product loop are
+			 * tax excluded.
+			 */
+			await WooCommerce.post( 'settings/tax/batch', {
+				update: [
+					{
+						id: 'woocommerce_tax_display_shop',
+						value: 'excl',
+					},
+					{
+						id: 'woocommerce_tax_display_cart',
+						value: 'incl',
+					},
+				],
+			} );
+			await shopper.emptyCart();
+		} );
+
+		afterAll( async () => {
+			const displayShop = taxSettings.find(
+				( setting ) => setting.id === 'woocommerce_tax_display_shop'
+			);
+			const displayCart = taxSettings.find(
+				( setting ) => setting.id === 'woocommerce_tax_display_cart'
+			);
+			await WooCommerce.post( 'settings/tax/batch', {
+				update: [
+					{
+						id: 'woocommerce_tax_display_shop',
+						value: displayShop.value,
+					},
+					{
+						id: 'woocommerce_tax_display_cart',
+						value: displayCart.value,
+					},
+				],
+			} );
+			await shopper.emptyCart();
+		} );
+
+		it( 'Mini Cart show tax label and price including tax', async () => {
+			const [ priceInLoop ] = await getTextContent(
+				'.wc-block-grid__product:first-child .wc-block-grid__product-price'
+			);
+
+			await page.click(
+				'.wc-block-grid__product:first-child .add_to_cart_button'
+			);
+
+			await expect( page ).toMatchElement( '.wc-block-mini-cart__title', {
+				text: 'Your cart (1 item)',
+			} );
+
+			await page.waitForSelector( '.wc-block-cart-item__prices' );
+			const [ priceInCart ] = await getTextContent(
+				'.wc-block-cart-item__prices'
+			);
+
+			expect( priceInLoop ).not.toMatch( priceInCart );
+
+			await page.mouse.click( 50, 200 );
```

Comment:
What's the purpose of this? To close the mini cart? In general, I'd advise against such low level implementation details in a test file. It might be tempting since a method would be a one liner, but `await miniCart.close()` reads much better than `await page.mouse.click( 50, 200 )` and will also allow us to more easily switch implementation (for example when we'll switch to another library or framework).

Reply:
This makes a lot of sense to me. I addressed this in [ff3ece3](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/5979/commits/ff3ece3d3702e9e700cf5fcc75952d178ade3701)